Storage/DataManager.php

```diff
@@ -106,6 +106,15 @@ public function load(FormFlowInterface $flow) {
 	/**
 	 * {@inheritDoc}
 	 */
+	public function hasInstance(FormFlowInterface $flow) {
+		// See if data for the given flow exists
```

Comment:
The intent of the comment was actually explaining what the definition of "exists" really is, though re-reading it now, it does seem ambiguous and redundant. I improved it.